endpoints/api/namespacequota.py

```diff
@@ -108,15 +108,13 @@ def get(self, orgname):
     @nickname("createOrganizationQuota")
     @validate_json_request("NewOrgQuota")
     @require_scope(scopes.SUPERUSER)
+    @show_if(features.SUPER_USERS)
```

Comment:
Might as well just put this at the class level, since it doesn't really make sense to have quota enabled, but not superusers in this case.

Likewise for the other resource classes.

Reply:
Ok, makes sense